modules/developer_manual/pages/core/apis/ocs-share-api.adoc

```diff
@@ -676,8 +676,11 @@ The permissions to set on the share.
 * 2 = update;
 * 4 = create;
 * 8 = delete;
-* 15 = read/write;
 * 16 = share;
+
+Common example combinations are:
+
+* 15 = read/write/delete;
```

Comment:
```suggestion
* 15 = read/write(update and create)/delete;
```

In this case "write" is a word for "update+create". I don't know if that should be made explicit here, so that readers do not wonder where "write" came from.